src/Screens/Versions.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import axios from "axios";
+import BASE_URL from "../API/BASE_URL";
+import Paginate from "../Components/Paginate"
+import Breadcrumbs from "../Components/Breadcrumbs";
+import { useHistory } from "react-router";
+import constant from "../Constants/EnglishScreens.json";
+import SimpleForm from "../Components/SimpleForm";
+import PaginateForm from "../Components/PaginateForm";
+import { PageSection, PageSectionVariants } from "@patternfly/react-core";
+
+export default function Versions(props) {
+  const [elementsRight, setElementsRight] = useState([]);
+  const [elementsLeft, setElementsLeft] = useState([]);
+  const [elements, setElements] = useState([]);
+  const [offset, setOffset] = useState(0);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(10);
+  const [productsVersion, setProductsVersion] = useState([]);
```

Comment:
You have set this multiple times? is that required?

Reply:
Yes, that's how Patternfly implemented the pagination, but if you have any better way then, please suggest.